zerver/views/registration.py

```diff
@@ -314,7 +313,7 @@ def send_registration_completion_email(email, request, realm_creation=False):
     """
     prereg_user = create_preregistration_user(email, request, realm_creation)
     activation_url = Confirmation.objects.get_link_for_object(prereg_user, host=request.get_host())
-    send_email('zerver/emails/confirm_registration', email, from_email=settings.DEFAULT_FROM_EMAIL,
```

Comment:
Same as above; isn't the confirmation link something nobody else should get access to?